echopype/commongrid/api.py

```diff
@@ -63,10 +66,18 @@ def _set_MVBS_attrs(ds):
 
 
 @add_processing_level("L3*")
-def compute_MVBS(ds_Sv, range_meter_bin=20, ping_time_bin="20S"):
+def compute_MVBS(
+    ds_Sv: xr.Dataset,
+    range_var: Literal["echo_range", "depth"] = "echo_range",
+    range_meter_bin: int = 20,
```

Comment:
I wonder if we can make this `range_bin` to parallel the new `range_var`, and have the input pattern be of the form "20m" to parallel the `ping_time_bin`, and we parse out the number 20 while do not allow other units (it does not make sense to do MVBS for km, and probably not cm in most cases).